tests/data/specs/array_message.rflx

```diff
@@ -18,4 +18,15 @@ package Array_Message is
          Bar : Bar;
       end message;
 
+   type Message_With_SDU is
+      message
+         Length : Byte
+            then Bar
+               with Size => Length * 8;
+         Bar : Bar
+            then Payload
+               with Size => 8;
+         Payload : Opaque;
+      end message;
```

Comment:
I don't like the `_With_SDU` suffix. `Bar` could be also seen as `SDU`, thus `Message` would be also a "message with SDU".

Reply:
Renames the messages in the `Array_Message` package in 55b15af